src/app/rundown-view/components/segment/segment.component.html

```diff
@@ -7,8 +7,11 @@
     <div class="c-segment__header-toolbar"
       [class.c-segment__header-toolbar--unsynced]="segment.isUnsynced"
     >
-      <span class="segment-identifier">
-        <ng-container *ngIf="segment.isUnsynced">UNSYNCED</ng-container>
+      <span class="segment-reference-tag">
+        <span *ngIf="!segment.isUnsynced else unsynced">{{segment.referenceTag}}</span>
```

Comment:
The `span` can be a `ng-container` to avoid having the extra element.

Reply:
No longer there after the changes.